tribe-ext-formatted-event-date-shortcode/index.php

```diff
@@ -31,25 +32,27 @@ class Tribe__Extension__Formatted_Event_Date_Shortcode extends Tribe__Extension
 	// Define the shortcode name
 	public $shortcode_name = 'tribe_formatted_event_date';
 	
+	// Define the timezone to display in the output, if any
+	private $timezone_for_display = '';
+	
 	/**
 	 * Setup the Extension's properties.
 	 *
 	 * This always executes even if the required plugins are not present.
-	 * 
-	 * @access: public
 	 */
 	public function construct() {
 		// Each plugin required by this extension
 		$this->add_required_plugin( 'Tribe__Events__Main', '4.0' ); // main thing we are concerned about is '_EventTimezone' meta key, but there are protections here against it not existing anyway
 		
 		// Set the extension's TEC URL
```

Comment:
Your code is beautifully self documenting. Ideally code is always self documenting, and thus doesn't need comments like this. If I were you I would remove redundant comments. 

Anytime you edit self documented code, you don't have to worry about updating the accompanying comments. But when you have a redundant comment, you must also worry about updating the comments to match the new code.